### ISSUE TITLE
UCP options not visible anymore

### ISSUE THREAD

**Issue by bug-i** [Sajaki/RecentTopics#93]
I am quite sure, that I have seen the UCP options in the past. Right now I do not see them anymore. I can set these options as admin in the backend, but cant find it as user within "user settings"

I have two different installations on different servers. Both phpBB 3.2.2. One of them is very clean without any other extensions and using prosilver. So I think this must be related to recent topics or one of the latest phpBB-updates. (3.2.1 to 3.2.2)  Could you check that on your installation(s)?

Recent topics is activated and visible in ACP, but not in UCP - see screenshots attached:

![acp](https://user-images.githubusercontent.com/29623218/36433070-fa853030-165b-11e8-9b26-c2793fc5ab31.jpg)

![ucp](https://user-images.githubusercontent.com/29623218/36433072-fae45948-165b-11e8-9fd7-16b343c18e1f.jpg)

**Comment by avandenberghe** [comment]
hello, 
The user should belong to a user group that has access to the UCP options. 
please see also the faq.  https://www.avathar.be/bbdkp/viewtopic.php?f=16&t=4064

**Comment by bug-i** [comment]
Hi Sajaki, thanks - but this solves unfortunately not everything. I managed by changing group-rights to show the options in the UCP, but there is no option for "Number of Recent topics to show" in the UCP. Also I cant find an option for "Number of Recent topics to show" to enable within the group rights. In the admin-backend "Number of Recent topics to show" has moved to UCP-section with the latest version on github. 

![admin](https://user-images.githubusercontent.com/29623218/36602936-9772141c-18b9-11e8-986e-6fc16f7d6f41.png)
![group rights](https://user-images.githubusercontent.com/29623218/36602937-978e5c30-18b9-11e8-9ea3-c594b05c64cb.png)

**Comment by avandenberghe** [comment]
fixed in release 2.2.7

**Comment by bug-i** [comment]
Almost fixed :) Still one minor bug:
The option in UCP "can change setting of number of recent topics to show per page." is only shown, if group rights "can enable or disable Displaying Recent Topics." are set to "yes". 
If "can enable or disable Displaying Recent Topics." is set to "no" - "can change setting of number of recent topics to show per page." is not shown. No matter if I set it to yes or no.

Another question: Whats the purpose behind the last option/group right "Recent Topics: can view Recent topics." - if I enable that in the group rights it doesnt change anything at all. No additional option in UCP is available, and recent topics is shown. No matter if I set it to yes or no. Kind of duplicate to the first option "can enable or disable Displaying Recent Topics" which works perfect.
